nntrainer/src/tensor.cpp

```diff
@@ -545,6 +545,18 @@ Tensor Tensor::sum(int axis, float alpha) const {
   return ret;
 }
 
+Tensor Tensor::sum(const std::vector<int> &axes, float alpha) const {
+  if (axes.empty() || axes.size() > MAXDIM)
+    throw std::invalid_argument("empty or too many axes given");
+
+  Tensor ret = this->sum(axes[0], alpha);
+
```

Comment:
It looks alpha scales multiple times. Could you check this?

Reply:
Right. It is fixed and I added a regression test.